core/arch/arm/plat-versal/link.mk

```diff
@@ -0,0 +1,11 @@
+include core/arch/arm/kernel/link.mk
+
+all: $(link-out-dir)/tee-raw.bin
```

Comment:
Shouldn't that be `all: $(link-out-dir)/uTee`? In other words, why should `tee-raw.bin` be generated by default but not `uTee`?

Reply:
sorry yes. we dont really need uTee

I think I can just 

`all: $(link-out-dir)/tee-raw.bin  `

Reply:
OK, so with just the above two lines (include + all:) this commit is:

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`